packages/salesforcedx-apex-debugger/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "rootDir": ".",
     "outDir": "out",
     "preserveConstEnums": true,
+    "skipLibCheck": true,
```

Comment:
This is required b/c the two version of TS don't play nice between this repo and SDR 
```
salesforcedx-vscode-core: node_modules/@salesforce/core/lib/sfdxError.d.ts(152,9): error TS2380: 'get' and 'set' accessor must have the same type.
salesforcedx-vscode-core: node_modules/@salesforce/core/lib/sfdxError.d.ts(153,9): error TS2380: 'get' and 'set' accessor must have the same type.```
are the kinds of errors found in several module.

Reply:
As discussed in standup, if there is no other way to specifically skip checks for selected libs, then we can go with this at the risk of skipping type checks and potentially introducing type errors until this is fixed. 
This adds more motivation to update typescript/tsconfig. 

Can you do a quick check on what happens if we update the ES target in tsconfig(s) in this repo/packages to match  the target libs?

Reply:
This article was pretty good for skipLibCheck: https://dd.engineering/blog/typescript-the-skiplibcheck-option-explained
I verified there is no option to specific it per module. 

I can try altering our target to match SDR. However I have low expectation b/c then we'll be different the node-apex (and some others I'm sure). That def feel like it belongs in the ts upgrade work vs here.

Reply:
attempted the updated and ran into the other way problem with `@salesforce/core` 
```
@salesforce/salesforcedx-utils-vscode: node_modules/@salesforce/core/lib/sfdxError.d.ts(152,9): error TS2380: 'get' and 'set' accessor must have the same type.
@salesforce/salesforcedx-utils-vscode: node_modules/@salesforce/core/lib/sfdxError.d.ts(153,9): error TS2380: 'get' and 'set' accessor must have the same type.
```